packages/edge-demo-site/app/entry.server.tsx

```diff
@@ -1,19 +1,28 @@
-import type { EntryContext } from '@remix-run/node'
+import type { EntryContext } from '@remix-run/server-runtime'
 import { RemixServer } from '@remix-run/react'
-import { renderToString } from 'react-dom/server'
+import { renderToReadableStream } from 'react-dom/server'
```

Comment:
For more about `renderToReadableStream`, see https://beta.reactjs.org/reference/react-dom/server/renderToReadableStream